apps/asap-server/test/handlers/user/index-handler.test.ts

```diff
@@ -0,0 +1,311 @@
+import Boom from '@hapi/boom';
+import { EventBridgeEvent } from 'aws-lambda';
+import { indexUserHandler } from '../../../src/handlers/user/index-handler';
+import { SquidexWebhookUserPayload } from '../../../src/handlers/user/invite-handler';
+import { UserEventType } from '../../../src/handlers/webhooks/webhook-user';
+import { getUserEvent, getUserResponse } from '../../fixtures/users.fixtures';
+import { algoliaSearchClientMock } from '../../mocks/algolia-client.mock';
+import { userControllerMock } from '../../mocks/user-controller.mock';
+
+describe('User index handler', () => {
+  const indexHandler = indexUserHandler(
+    userControllerMock,
+    algoliaSearchClientMock,
+  );
+
+  afterEach(() => jest.clearAllMocks());
+
+  test('Should fetch the user and create a record in Algolia when the user is created', async () => {
+    const userResponse = getUserResponse();
+    userControllerMock.fetchById.mockResolvedValueOnce(userResponse);
+
+    await indexHandler(createEvent('user-1234'));
```

Comment:
Maybe make the id of `createEvent` optional so it is not necessary here? It looks like this exact value is required, while it can be anything.

When looking at this, I thought the id in `userResponse` is `user-1234`.

Reply:
done mate